extensions/CheckUser/CheckUser_body.php

```diff
@@ -319,22 +326,22 @@ protected function noMatchesMessage( $userName, $checkLast = true ) {
 			if ( $user_id ) {
 				$revEdit = $dbr->selectField( 'revision',
```

Comment:
Does `selectField()` imply `LIMIT 1`? Considering `$lastEdit = max( $revEdit, $logEdit );` one is all we need.